scripts/monitor/rano_monitor/widgets/tarball_subject_view.py

```diff
@@ -55,9 +49,15 @@ def on_mount(self):
     def update_status(self):
         tumor_status = self.query_one(".tumor-status", Static)
         brain_status = self.query_one(".brain-status", Static)
-        if self.__tumor_has_been_finalized():
+
+        id, tp = self.subject.split("|")
+        finalized_tumor_path = os.path.join(self.contents_path, id, tp, "finalized")
```

Comment:
This finalized tumor path differs from what is building at `get_finalized_tumor_path`, is it expected?

Reply:
This is expected, because the tarball version of the monitoring tool doesn't interact with the same paths, but the ones related to the tarball file